website_sale_product_detail_attribute_image/views/templates.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <template
+        id="product_detail_attribute_image"
+        inherit_id="website_sale.product"
+        active="True"
+        customize_show="True"
+        name="Attribute's Images"
+    >
+        <xpath
+            expr="//div[@id='product_details']/../div[@class='col-md-6 col-xl-8']"
```

Comment:
Reading runbot's logs you can find this warning:

```
2020-06-05 08:11:00,683 170 INFO openerp_test odoo.modules.loading: loading website_sale_product_detail_attribute_image/views/templates.xml
2020-06-05 08:11:00,702 170 WARNING openerp_test odoo.addons.base.models.ir_ui_view: Error-prone use of @class in view Attribute's Images (): use the hasclass(*classes) function to filter elements by their classes
```

Runbot gets an :x: if installing the module raises a warning. Here's the problem.  Fix:

```suggestion
            expr="//div[@id='product_details']/../div[hasclass('col-md-6', 'col-xl-8')]"
```

Reply:
Thanks for the help! I change it yet